.gitignore

```diff
@@ -88,12 +88,16 @@ qrc_*.cpp
 # Qt creator
 *.pro.user
 
+# NetBeans
```

Comment:
I'm currently not at home, but later I can edit and remove the lines. But just for my understanding: isn't it pretty common to ignore project/editor specific files via the `.gitignore`?

As for the change:
```diff
- # Qt creator
- *.pro.user
- 
- # NetBeans
- nbproject/
- 
```
Anything else?